app/controllers/forms/make_live_controller.rb

```diff
@@ -1,8 +1,11 @@
 module Forms
   class MakeLiveController < BaseController
     def new
-      redirect_to live_confirmation_url if current_form.live? && !FeatureService.enabled?(:draft_live_versioning)
-      @make_live_form = MakeLiveForm.new(form: current_form)
+      if current_form.live? && !FeatureService.enabled?(:draft_live_versioning)
+        redirect_to live_confirmation_url
+      else
+        render template: "forms/make_live/new", locals: { make_live_form: MakeLiveForm.new(form: current_form) }
+      end
```

Comment:
Am I right in thinking this code is still doing the same thing after this change, just more explicitly?

Reply:
Once draft/live is finished and we remove the flag I dont think there will be a redirect at all. 
We could have a redirect in this controller action if all the required fields are completed or there aren't any changes but lets keep it simple for the time being and extend on.